lib/messages.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   userConfig: (path) => `Couldn't read the userConfig from ${path}!`,
-  breakpoints: (path) => `Couldn't read drupals breakpoints file from ${path}!`,
+  breakpoints: (path) =>
+    `Couldn't read drupal's breakpoints file from ${path}!`,
   writeCSSFile: (path) => `Couldn't write css file to ${path}!`,
   writeJSFile: (path) => `Couldn't write js file to ${path}!`,
```

Comment:
The Drupal community probably wants Drupal to be written with an uppercase D, like a proper noun. Same for “CSS” and “JS” 😅